src/Core/Utilities/CoreHelpers.cs

```diff
@@ -459,6 +459,7 @@ public static string FormatLicenseSignatureValue(object val)
             return val.ToString();
         }
 
+        [Obsolete("CoreHelpers.GetVersion is deprecated, please use AssemblyHelpers.GetVersion instead.")]
```

Comment:
Can we just remove it completely instead?

Reply:
Removed 👍